src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -84,3 +55,27 @@ new Replay({
   // any further configuration here is passed directly to rrweb
 });
 ```
+
+## Start and Stop Recording
+
+Replay recording only starts automatically when it's included in the integrations key when calling Sentry.init. Otherwise you can initialize the plugin and manually call the start() method on the integration instance. To stop recording you can call the stop():
```

Comment:
```suggestion
Replay recording only starts automatically when it's included in the integrations key when calling `Sentry.init`. Otherwise, you can initialize the plugin and manually call the `start()` method on the integration instance. To stop recording you can call the `stop()`:
```